content/creator/worlds/about.md

```diff
@@ -97,7 +125,7 @@ Once you run the command, you will be prompted to sign the deployment with your
 ## Joining a World
 
 Once a scene is uploaded to the Worlds server you can access it by using the
-Decentraland Explorer with the following friendly URL `https://play.decentraland.org/world/NAME`, where `NAME` should be replaced by the Decentraland NAME to which the deployment was done to. You may use the NAME with or without the `.dcl.eth` suffix.
+Decentraland Explorer with the following friendly URL `https://play.decentraland.org/world/NAME`, where `NAME` should be replaced by the Decentraland NAME or ENS Domain to which the deployment was done to. You may use the NAME with or without the `.dcl.eth`, `.eth` suffixes.
```

Comment:
```suggestion
Decentraland Explorer with the following friendly URL `https://play.decentraland.org/world/NAME`, where `NAME` should be replaced by the Decentraland NAME or ENS Domain to which the deployment was done to. You may use the NAME with or without the `.dcl.eth` suffix. If no suffix is added, it will assume `.dcl.eth`.
```